site/docs/v1/tech/lambdas/index.adoc

```diff
@@ -79,21 +175,18 @@ function populate(jwt, user, registration) {
 }
 ----
 
-=== Engine
-
-Nashorn is built on top of the Java virtual machine and while Nashorn permits access to the Java API, for security reasons FusionAuth restricts access to all Java objects during a lambda invocation. Here is the documentation provided by Oracle for the Nashorn engine:
-
-https://docs.oracle.com/javase/8/docs/technotes/guides/scripting/nashorn/
-
 == Limitations
 
 If the Identity Provider linking strategy is set to `Link Anonymously`, no lambdas will be used by FusionAuth. More information about the link:/docs/v1/tech/identity-providers/#linking-strategies[Identity Provider linking strategies is available here].
 
-The FusionAuth lambdas do not have full access to JavaScript modules and libraries. They also cannot import, require or load other libraries currently. These features might be added to our lambda support in the future but Nashorn does not support them currently.
+The FusionAuth lambdas do not have full access to JavaScript modules and libraries. They also cannot import, require or load other libraries currently. These features might be added to our lambda support in the future.
 
 `console.log` and other `console` methods only take one argument; this differs from the `console` method available in web browsers.
 
-Only the populate or reconcile function is allowed on the top level of a lambda. If you want to have additional functions, create them as function objects inside the main function. For example:
+Only the populate or reconcile function is allowed on the top level of a lambda.
+If you want to have additional functions, create them as function objects inside the main function.
```

Comment:
Are you sure, I just did a test and I could call functions defined within the main function, and also those defined anywhere else in the lambda.

Reply:
Hmmm. I thought this advice came from user experience/bug but maybe that is a nashorn limitation? Will check further.

Reply:
Tested and having functions defined elsewhere works in both engines. Will change.